front/src/modules/ui/table/components/DropdownComponents.tsx

```diff
@@ -0,0 +1,82 @@
+import { StyledDropdownMenu } from '@/ui/dropdown/components/StyledDropdownMenu';
+import { StyledDropdownMenuItemsContainer } from '@/ui/dropdown/components/StyledDropdownMenuItemsContainer';
+import { useDropdownButton } from '@/ui/dropdown/hooks/useDropdownButton';
+import { ViewFieldMetadata } from '@/ui/editable-field/types/ViewField';
+import { IconArrowLeft, IconArrowRight, IconEyeOff } from '@/ui/icon';
+import { MenuItem } from '@/ui/menu-item/components/MenuItem';
+
+import { ColumnHeadDropdownId } from '../constants/ColumnHeadDropdownId';
+import { useTableColumns } from '../hooks/useTableColumns';
+import { ColumnDefinition } from '../types/ColumnDefinition';
+
+export type EntityTableHeaderOptionsProps = {
+  column: ColumnDefinition<ViewFieldMetadata>;
+  isFirstColumn: boolean;
+  isLastColumn: boolean;
+  primaryColumnKey: string;
+};
+
+export const DropdownComponents = ({
```

Comment:
`TableColumnDropdownMenu`

Reply:
It's important to avoid generic names like this one and add the relevant context in the naming, especially since we are using a lot of dropdowns in the app.

Reply:
That's fine @lucasbordeau, we'll avoid doing this.

Thanks for the review

Reply:
Hey @lucasbordeau,
The issues mentioned above have been resolved.